releasenotes/notes/asm-iast-report-scrubbing-ef2aa1d74f866656.yaml

```
@@ -0,0 +1,4 @@
+---
+other:
```

```suggestion
features:
```

Could we get a bit more information on how this can be configured/used? A link to the configuration docs would also be helpful as well.